scripts/install_chrome_on_travis.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script should only be ran by Travis to install and provide a constant
+version of Chrome.
+"""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import argparse
+import os
+import subprocess
+
+import python_utils
+
+_PARSER = argparse.ArgumentParser(description="""
+This script should only be ran by Travis to install and provide a constant
```

Comment:
```suggestion
This script should only be run by Travis to install and provide a constant
```

Reply:
Also, shouldn't these lines ideally be indented by two? I think you can use single quote strings instead of """ as we typically use the latter for docstrings.

Reply:
Done and used single quotes

Reply:
The description is exceptional since it typically has a ton of information to give. Also, indents show up in the command line and make the output look weird.

Reply:
I tested with single quotes and the indent does not get messed up. So should I still revert the last commit?

Reply:
Also isn't it better to have a use only one type of quotes throughout all descriptions, just to maintain consistency

Reply:
I agree, having the description break at 80-characters is a nice plus too for very long sentences (which will not happen here).

I vote for triple-quote strings and no indentations, what do you think @nithusha21?